app/views/layouts/application.html.haml

```diff
@@ -1,58 +1,70 @@
 !!! 5
+<!DOCTYPE html>
```

Comment:
I probably wanted to remove one of those...
https://affotd.files.wordpress.com/2013/09/huge-mistake-arrested-development.gif